content/getting-started/authorization/index.mdx

```diff
@@ -456,6 +457,29 @@ Private apps can be used only on accounts chosen by the developer.
 
 This authorization flow is not yet available.
 
+## Personal Access Tokens
+
+Unlike other authorization flows that let you acquire a token for your app, **Personal Access Tokens (PATs)** are generated per Agent.
+
+With the use of PATs, you can make requests to the Agent Chat and Configuration API but not to the Customer Chat API.
```

Comment:
Pozwala też robić requesty do Reports API, i pewnie gdzieś jeszcze. Generalnie do większości API, nie pisałbym że Customer Chat API jest wykluczone.

Reply:
Ok. The argument that someone may CTRL + F and find Customer API convinced me. 
Added Reports API.